cli/build.gradle

```diff
@@ -172,6 +172,17 @@ task unzipPortal(dependsOn: downloadPortal, type: Copy) {
 	from zipTree(downloadPortal.dest)
 	into buildDir
 }
+task cliTestDependencies(type: Jar) {
```

Comment:
Declare tasks at the top

`task testJar(type: Jar)`

and then later sorted alphabetically declare the task bode
```
testJar {
    from sourceSets.test.output
    classifer = "test"
}
```